src/App.jsx

```diff
@@ -0,0 +1,107 @@
+import './App.css';
+import {Link} from 'react-router-dom';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+
+import Header from './component/Header';
+import BestProducts from './component/BestProducts';
+import ProductList from './component/ProductList';
+import Pagination from './component/Pagination';
+import SortMenu from './component/SortMenu';
+
+function App() {
+  const [bestProducts, setBestProducts] = useState([]); 
+  const [isOpen, setOpen] = useState(false);
+  const [sortbtntext, setSortbtntext] = useState('최신순');
+  const [sortedProducts, setSortedProducts] = useState([]);
+  const pagebuttons = [1, 2, 3, 4, 5];
```

Comment:
P2:
react에서 변수명은 일반적으로 카멜케이스로 작성됩니다. 
아래와 같은 방식을 권합니다.

가능하다면 button말고 다른 이름이 좋을 것 같습니다.

```suggestion
  const pageButtons = [1, 2, 3, 4, 5];
```